packages/@adobe/spectrum-css-temp/components/menu/skin.css

```
@@ -51,7 +51,7 @@ governing permissions and limitations under the License.
     background-color: var(--spectrum-selectlist-option-background-color-down);
   }
 
-  &.is-disabled {
+  &.is-disabled, &.is-disabled .spectrum-Menu-description {
```

this is applying a lot of things that I don't think are needed. we should minimize the properties we set
```suggestion
  &.is-disabled {
    .spectrum-Menu-description {
      color: var(--spectrum-selectlist-option-text-color-disabled);
    }
```